common/lib/xmodule/xmodule/js/src/word_cloud/word_cloud_main.js

```diff
@@ -12,106 +12,110 @@
  */
 
 (function(requirejs, require, define) {
-    define('WordCloudMain', [], function() {
-    /**
-     * @function WordCloudMain
-     *
-     * This function will process all the attributes from the DOM element passed, taking all of
-     * the configuration attributes. It will either then attach a callback handler for the click
-     * event on the button in the case when the user needs to enter words, or it will call the
-     * appropriate mehtod to generate and render a word cloud from user's enetered words along with
-     * all of the other words.
-     *
-     * @constructor
-     *
-     * @param {jQuery} el DOM element where the word cloud will be processed and created.
-     */
+    'use strict';
+    define('WordCloudMain', [
+        'gettext',
+        'edx-ui-toolkit/js/utils/html-utils'
+    ], function(gettext, HtmlUtils) {
+        function generateUniqueId(wordCloudId, uniqueId) {
```

Comment:
I mean that if you remove the random generation of uniqueId and simply use a counter "uniqueId" is no longer unique. "index" would be a more appropriate name.